prboom2/src/SDL/i_sound.c

```diff
@@ -1504,6 +1508,7 @@ static int Exp_RegisterSongEx (const void *data, size_t len, int try_mus2mid)
           }
           else
             lprintf (LO_INFO, "Exp_RegisterSongEx: Music player %s on preferred list but it failed to init\n", music_players[i]-> name ());
+          break;
```

Comment:
Why do you introduce this `break` here?

Reply:
Because the following iterations of this loop are wasted. It doesn't matter much, but it was a bit annoying when debugging with gdb. :p